src/360-core/browzine-360-core-adapter.js

```diff
@@ -159,7 +159,15 @@ browzine.serSol360Core = (function() {
         }
 
         if(coverImageUrl && browzineEnabled) {
-          $(title.target).find("img.results-title-image").attr("src", coverImageUrl).attr("ng-src", coverImageUrl).css("box-shadow", "1px 1px 2px #ccc");
+          var resultsTitleImageSelector = ".results-title-image-div img.results-title-image";
+          var boxShadow = "1px 1px 2px #ccc";
+
+          if($(title.target).find(resultsTitleImageSelector).length > 0) {
+            $(title.target).find(resultsTitleImageSelector).attr("src", coverImageUrl).attr("ng-src", coverImageUrl).css("box-shadow", boxShadow);
+          } else {
+            $(title.target).find(".results-title-image-div").append("<img class='results-title-image'/>");
```

Comment:
_non-blocking PR comment_

Want to name this selector on a variable to describe what it is, too?